straxen/analyses/bokeh_waveform_plot.py

```diff
@@ -15,6 +15,7 @@ def event_display_interactive(events, to_pe, run_id, context, xenon1t=False):
     Interactive event display for XENONnT. Plots detailed main/alt
     S1/S2, bottom and top PMT hit pattern as well as all other peaks
     in a given event.
+    If used with 1T data xenon1t flag must be True.
```

Comment:
I was actually referring to the :
```
  > straxen.event_display(context,
        >                       run_id,
        >                       time_range=(event['time'],
        >                                   event['endtime'])
        >                      )
```

It should be from the context and the function is not this wf plot

Reply:
Ah yes :D